src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1819,4 +1798,14 @@ void PolicyHandler::Add(const std::string& app_id,
   policy_manager_->Add(app_id, type, timespan_seconds);
 }
 
+bool PolicyHandler::IsUrlAppIdValid(const std::string& policy_app_id) const {
+  ApplicationSharedPtr app =
+      application_manager_.application_by_policy_id(policy_app_id);
+
+  const bool is_registered = (app && (app->IsRegistered()));
+  const bool is_default = (policy_app_id == policy::kDefaultId);
+
+  return (is_registered || is_default);
```

Comment:
@tkireva Why didn't you check  has_urls = !urls.at(current_app).url.empty(); in new method?